src/ALCProxy.Communication/ALCProxy.Communication.csproj

```diff
@@ -0,0 +1,10 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup> 
+   <Description>Non-allocating collections</Description>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks> 
+   <PackageTags>collections, non-allocating, corefxlab</PackageTags> 
```

Comment:
nit: fix indentation. Looks like other projects are using 2 spaces.

PackageTags should only be relevant if we're going to publish this as a package. Seems like the repo packaging logic may pick up all projects: https://github.com/dotnet/corefxlab/blob/7076498fd58574e01bbc6747a788a235bbcc9217/scripts/package.ps1 - we should make sure this doesn't get packaged unless it's intentional.